### PR TITLE
Known issues

### PR DESCRIPTION
**Pull request by JorisGoosen**
Should be merged AFTER https://github.com/jasp-stats/jasp-desktop/pull/4240 !

Screenshot_2020-08-18_130523
Screenshot_2020-08-18_130534

### CODE REVIEW

**Comment by JorisGoosen** [comment]
Maybe I need to rebase the branch first after https://github.com/jasp-stats/jasp-desktop/pull/4240 is merged because it seems those commits are also in here...

**Comment by EJWagenmakers** [comment]
Can we replace the warning text with this:
"WARNING. A bug has been reported for this analysis. Click on this text for details."
Hmm I am not too happy with that either...How about:
"WARNING. This analysis contains an error, and we are in the process of fixing it. Click on this text for details. Proceed at your own risk."
?!

**Comment by JorisGoosen** [comment]
The warning text is part of the json file on the server so it can be tweaked to your heart's desire at any time. The one in the screenshot is just to show the mechanism

**Comment by JorisGoosen** [comment]
So Ive updated this now that the errormessages PR was merged, so this can be checked and merged if given the OK.

File: JASP-Desktop/gui/jaspversionchecker.cpp [hunk 1]
```diff
@@ -47,6 +56,14 @@ void JASPVersionChecker::downloadFinished()
 	if (latest > cur)
 		emit showDownloadButton(downloadfile);
 
+	if(KnownIssues::issues()->downloadNeededOrLoad())	downloadKnownIssues();
```
Comment (boutinb) [inline JASP-Desktop/gui/jaspversionchecker.cpp:59]: This is double: downloadKnownIssues is already called in downloadNeededOrLoad

File: JASP-Desktop/gui/jaspversionchecker.cpp [hunk 2]
```diff
@@ -47,6 +56,14 @@ void JASPVersionChecker::downloadFinished()
...
+	else												deleteLater(); //Remove yourself!
+}
```
Comment (boutinb) [inline JASP-Desktop/gui/jaspversionchecker.cpp:61]: I would set a JASPVersionChecker as member mainWindow and a add a method to start the check. This deleteLater while waiting for a network response is a bit scary... and is not really needed.

**Review by boutinb — CHANGES_REQUESTED** [review]

**Comment by boutinb** [comment]
The cross to remove the warning does not work.

File: JASP-Desktop/gui/jaspversionchecker.cpp (hunk 1 above)
Comment (JorisGoosen) [reply JASP-Desktop/gui/jaspversionchecker.cpp:59]: No it isn't.

Show me where then?

File: JASP-Desktop/gui/jaspversionchecker.cpp (hunk 2 above)
Comment (JorisGoosen) [reply JASP-Desktop/gui/jaspversionchecker.cpp:61]: The deleteLater isn't called while waiting for a network response...

**Comment by JorisGoosen** [comment]
Aah yeah, well, I think it is good to have the warning permanently there. Because they are known issues after all and I want people to be aware of them.

**Comment by JorisGoosen** [comment]
Hmm, but it is more difficult to do now that they aren't cleared all the time. I'll make sure the cross works. But then the warnings on the options should probably also be removed.

**Review by boutinb — APPROVED** [review]

**Pull request merged**